core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/IlluminaRun.java

```diff
@@ -98,4 +99,17 @@ public PlatformType getPlatformType() {
     return PlatformType.ILLUMINA;
   }
 
+  @Override
+  public String getProgress() {
+    if (getHealth() == HealthType.Running) {
+      if (numCycles != null && callCycle != null) {
+        if (numCycles != 0) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1066)